CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 - The tour is generated through an independent module and steps are chosen depending the available modules. See issue [#166](https://github.com/mitre-attack/attack-website/issues/166).
 - Modules and test arguments are now required if flags are presented. See issue [#177](https://github.com/mitre-attack/attack-website/issues/177).
 - Updated website dependencies. See issue [#181](https://github.com/mitre-attack/attack-website/issues/181).
+- Removed the PRE-ATT&CK domain, including its matrix, techniques/tactics listings, and references to the domain. See issue [#222](https://github.com/mitre-attack/attack-website/issues/222).
```

Comment:
Revise changelog to:
> Removed PRE-ATT&CK domain to support migration into the new (TODO: list and link to tactics) tactics in Enterprise-ATT&CK.  See issue [#222](https://github.com/mitre-attack/attack-website/issues/222).

This change is necessary because from a user's perspective pre-ATT&CK is migrating to new tactics, and we want to make it apparent that it's not gone entirely.

Reply:
The TODO should remain a TODO in the actual changelog, we can fill in that section right before the release.

Reply:
Sounds good! I fixed up the changelog with your suggestion.